benchmarks/oscilator_integrator_order/integrator_order_experiment.m

```diff
@@ -64,7 +65,7 @@
     solver_options.complementarity_tol = complementarity_tol;
     solver_options.N_homotopy = 40; % number of steps
     solver_options.homotopy_update_slope = 0.05; % decrease rate
-    solver_options.use_previous_solution_as_initial_guess = 1; % warm start integrator
+    % solver_options.use_previous_solution_as_initial_guess = 1; % warm start integrator
```

Comment:
this lives now in integrator options and causes a bug. this bug is present in most examples where this option is changed

Reply:
Changed in examples affected by this.